python/amici/pysb_import.py

```diff
@@ -10,6 +10,10 @@
     Expression, LogLikelihood, generate_measurement_symbol
 )
 
+from .sbml_import import (
+    _parse_special_functions
+)
```

Comment:
Let's move that function to `.import_utils` instead of pulling SBML stuff into pysb import.